### PR TITLE
Updatefix

### PR DESCRIPTION
**Pull request by qimingj**

### CODE REVIEW

File: content/publish.sh [hunk 1]
```diff
@@ -14,9 +14,13 @@
 # limitations under the License.
 
 # Publishes content files to Google Cloud Storage as publicly accessible content.
-
 # Synchronize content from the repository content directory to the
 # cloud within gs://cloud-datalab/content
+
+# First delete all content because rsync may not always override destination objects.
+gsutil -m rm -r -f gs://cloud-datalab/content
```
Comment (nikhilk) [inline content/publish.sh]: Can we just change the rsync to a cp, rather than trying to empty the bucket first?

**Comment by nikhilk** [comment]
Coment on the publish script, but LGTM on the actual fix.

File: content/publish.sh (hunk 1 above)
Comment (qimingj) [inline content/publish.sh]: Did some experiments and it seems cp does not delete objects that we want to remove. Emptying the bucket seems cleaner?

Comment (nikhilk) [inline content/publish.sh]: We really don't want to ever delete (since we can't delete from the users' repos anyway)

Comment (qimingj) [inline content/publish.sh]: I'll merge the update fix and create a new one on the publish fix.

**Pull request closed without merging**